Packs/PrismaAccess/Integrations/PrismaAccessEgressIPFeed/PrismaAccessEgressIPFeed.py

```diff
@@ -17,14 +17,16 @@ class Client(BaseClient):
     https://docs.paloaltonetworks.com/prisma/prisma-access/prisma-access-panorama-admin/prisma-access-overview/retrieve-ip-addresses-for-prisma-access
     """
 
-    def __init__(self, clientConfigs: list, api_key: str, insecure: bool = False, proxy: bool = False):
+    def __init__(self, clientConfigs: list, api_key: str, insecure: bool = False, proxy: bool = False,
+                 tags: Optional[list] = None):
```

Comment:
let's have the default value an empty list, and then:
self.tags = tags
```suggestion
                 tags: Optional[list] = []):
```